### PR TITLE
[infra] Various Wireit config improvements

### PR DESCRIPTION
**Pull request by aomarks**
- Turns out our base rollup config automatically also copies test HTML files as a side-effect. This was not accounted for in the Wireit config. Also, we were copying HTML files even in the checksize script, which we now no longer do.

- Added Wireit configs for benchmarks and localize examples.

- Expanded Wireit configs for tests and virtualizer. Now they can be cached.

- Added some missing dependencies and outputs.

- Upgrade to Wireit 0.7.1 to pick up the 503 error recovery fix (https://github.com/google/wireit/pull/326).

### CODE REVIEW

File: packages/localize/examples/runtime-ts/package.json [hunk 1]
```diff
@@ -5,9 +5,18 @@
...
     "serve": "web-dev-server --node-resolve"
   },
+  "wireit": {
+    "build": {
+      "command": "rm -rf lib/* && lit-localize build && tsc",
+      "dependencies": [
+        "../../../localize:build",
+        "../../../localize-tools:build"
+      ]
+    }
```
Comment (augustjk) [inline packages/localize/examples/runtime-ts/package.json:18]: Maybe not worth the effort but should this also have files and output specified like mentioned above? Including `lib` in output should also remove the need for the `rm -rf`.

**Review by justinfagnani — APPROVED** [review]
LGTM, pending @augustjk's comments!

File: packages/localize/examples/runtime-js/package.json [hunk 2]
```diff
@@ -5,9 +5,18 @@
...
+    "build": "wireit",
     "serve": "web-dev-server --node-resolve"
   },
+  "wireit": {
+    "build": {
+      "command": "lit-localize build",
+      "dependencies": [
+        "../../../localize:build",
+        "../../../localize-tools:build"
+      ]
```
[the remark this answers is not in the document]
Comment (aomarks) [reply packages/localize/examples/runtime-js/package.json]: Done

File: packages/localize/examples/runtime-ts/package.json [hunk 3]
```diff
@@ -6,14 +6,45 @@
...
+        "!src/generated/**"
+      ],
+      "output": [
+        "src/generated/**"
+      ]
+    },
+    "build:ts": {
+      "command": "tsc",
+      "dependencies": [
+        "build:localize",
```
Comment (augustjk) [inline packages/localize/examples/runtime-ts/package.json:40]: Oooh neat! I was about to comment for `build`'s dependency if the dependencies there need to be serial but this takes care of that.

File: packages/localize/examples/runtime-ts/package.json [hunk 4]
```diff
@@ -6,14 +6,45 @@
...
+        "xliff/**",
+        "!src/generated/**"
+      ],
+      "output": [
+        "src/generated/**"
+      ]
+    },
+    "build:ts": {
+      "command": "tsc",
+      "dependencies": [
```
Comment (augustjk) [inline packages/localize/examples/runtime-ts/package.json:39]: Nit. Should the clean option be added?
```suggestion
      "command": "tsc",
      "clean": "if-file-deleted",
      "dependencies": [
```

**Review by augustjk — APPROVED** [review]

**Review by justinfagnani — APPROVED** [review]

**Review by augustjk — APPROVED** [review]

File: rollup-common.js [hunk 5]
```diff
@@ -394,9 +395,9 @@ export function litProdConfig({
         sourcemaps(),
         terser(terserOptions),
         summary(),
-        ...(CHECKSIZE
-          ? [skipBundleOutput]
-          : [
+        ...(CHECKSIZE ? [skipBundleOutput] : []),
+        ...(copyHtmlTests && !CHECKSIZE
```
Comment (augustjk) [inline rollup-common.js:399]: I think parens is needed for this to be evaluated together before the ternary.
```suggestion
        ...((copyHtmlTests && !CHECKSIZE)
```

**Review by augustjk — CHANGES_REQUESTED** [review]

File: rollup-common.js (hunk 5 above)
Comment (augustjk) [reply rollup-common.js:399]: Nevermind, it seems parens isn't needed.

**Review by augustjk — APPROVED** [review]

File: packages/localize/examples/runtime-ts/package.json (hunk 4 above)
Comment (aomarks) [reply packages/localize/examples/runtime-ts/package.json]: Done (also set up incremental build, since `if-file-deleted` is pointless otherwise)

**Comment by aomarks** [comment]
Also threw in a bump to wireit to pick up the 503 error recovery fix.

**Pull request merged**